package.json

```diff
@@ -41,7 +41,7 @@
     "safe-json-parse": "4.0.0",
     "static-config": "2.1.0",
     "tape-cluster": "2.0.1",
-    "tchannel": "3.2.3",
+    "tchannel": "3.3.1",
```

Comment:
this test change is depend on a tchannel change:
https://github.com/uber/tchannel-node/pull/33/files#diff-e3ba5987085d79033122a81a2ddc357cR385

the tchannel change exposed an extra parameter consumed by the test.

without the tchannel change being upgraded, the test uses some parameter which doesn't exist, and therefore fails.